r/src/R/H2OContext.R

```diff
@@ -0,0 +1,70 @@
+getClientConnectedField <- function(jhc) {
```

Comment:
These are not in the class on purpose as we do not want them to be visible by the user